bin/bossSwitch.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3.5
+
+# Copyright 2016 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script to turn the boss on and off.
+   The script aims at switch the boss on and off once it has been configured a first time.
+   The instances should first be stood up by building stacks through cloudformation.
+   ASG are shut down. Only execute this code if you are certain the boss will not
+   be running for another hour."""
+
+import boto3
+import sys
+import json
+import argparse
+import os
+import subprocess
+import time
+import pickle
+
+import alter_path
+from lib import aws, utils, vault
+
+def main():
+
+    choice = utils.get_user_confirm("Are you sure you want to proceed switching the boss on/off?")
+    if choice:
+        if args.action == "on":
+            print("Turning the BossDB on...")
+            startInstances()
+
+        elif args.action == "off":
+            print("Turning the BossDB off...")
+            stopInstances()
+
+#Executed actions
+def startInstances():
+    """
+        Method used to start necessary instances
+    """
+    #Use auto scaling groups last saved configuration
+    ASGdescription = load_obj('ASGdescriptions')
+    activitiesD = [ASGdescription["AutoScalingGroups"][0]["MinSize"], ASGdescription["AutoScalingGroups"][0]["MaxSize"],ASGdescription["AutoScalingGroups"][0]["DesiredCapacity"]]
+    endpointD = [ASGdescription["AutoScalingGroups"][1]["MinSize"], ASGdescription["AutoScalingGroups"][1]["MaxSize"],ASGdescription["AutoScalingGroups"][1]["DesiredCapacity"]]
+    vaultD = [ASGdescription["AutoScalingGroups"][2]["MinSize"], ASGdescription["AutoScalingGroups"][2]["MaxSize"],ASGdescription["AutoScalingGroups"][2]["DesiredCapacity"]]
+   
+    #Start vault instance
+    print("Starting vault...")
+    client.update_auto_scaling_group(AutoScalingGroupName=vaultg, MinSize = vaultD[0] , MaxSize =  vaultD[1], DesiredCapacity = vaultD[2])
+    client.resume_processes(AutoScalingGroupName=vaultg,ScalingProcesses=['HealthCheck'])
+    time.sleep(120)
+    print("Vault instance running")
+
+    #Import vault content:
+    subprocess.call('./bastion.py ' + args.vpc + ' vault-unseal',shell=True)
+    print("Importing vault content")
+    subprocess.call('./bastion.py ' + args.vpc + ' vault-import < ../config/vault_export.json', shell=True)
+    print(bcolors.WARNING + "Successful import" + bcolors.ENDC)    
```

Comment:
It might make sense to create a `lib/` file for writing colored messages to the screen. It could be something like `console.warning("Successful import")`

Reply:
I think I'll write another function in `lib/utils.py` that handles color messaging